### ISSUE TITLE
Keypairs with a space in the name cause elektra to crash

### ISSUE THREAD

**Issue by SuperSandro2000** [sapcc/elektra#1369]
I've uploaded a keypair to the cloud via ``openstack keypair create "Sandro WSL SAP" --public-key ~/.ssh/id_ed25519.pub``. When I want to view that key, I get the following "500"

![image](https://github.com/sapcc/elektra/assets/7258858/4f60e253-670c-41f4-add1-864fcccdca12)

**Comment by edda** [comment]
This error message indicates that it's a 500 in the backend service, not in Elektra. What happens when you try to view the keypair using the OpenStack CLI?

**Comment by SuperSandro2000** [comment]
That works as expected:

```
 ▶ openstack keypair list
+----------------+-------------------------------------------------+------+
| Name           | Fingerprint                                     | Type |
+----------------+-------------------------------------------------+------+
| Sandro XXX XXX | xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx | ssh  |
+----------------+-------------------------------------------------+------+

 ▶ openstack keypair show "Sandro XXX XXX"
+-------------+------------------------------------------------------------------+
| Field       | Value                                                            |
+-------------+------------------------------------------------------------------+
| created_at  | 2024-06-17T11:22:33.000000                                       |
| fingerprint | xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx                  |
| id          | Sandro XXX XXX                                                   |
| is_deleted  | False                                                            |
| name        | Sandro XXX XXX                                                   |
| private_key | None                                                             |
| type        | ssh                                                              |
| user_id     | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+-------------+------------------------------------------------------------------+
```

**Comment by hgw77** [comment]
digged deeper and found the problem, it seems the view cannot handle spaces in the name

**Comment by hgw77** [comment]
this issue is also related to #176

**Comment by hgw77** [comment]
this should be fixed with #1382
